### PR TITLE
My Resume

### PR DESCRIPTION
**Pull request by ibomiddleton**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,8 +12,64 @@
...
+    <header>
+      <h1>Isaac Middleton - Web Dev</h1>
+      <nav>
+          <a href="#">Home</a>
+          <a href="#">About</a>
+          <a href="#">Portfolio</a>
+          <a href="#">References</a>
+      </nav>
+    </header>
+      <img src="images/pic06.jpg" alt="#" />
```
Comment (ari-mercado) [inline index.html:26]: Your images folder is capitalized "Images". It's generally a good idea to include a very brief description in the "alt". This gives screen readers information about the image.

File: index.html [hunk 2]
```diff
@@ -12,8 +12,64 @@
...
+      </nav>
+    </header>
+      <img src="images/pic06.jpg" alt="#" /> 
+    <div class="main-content">
+      <section class="intro">
+        <h2>About Me</h2>
+        <p>I am a student of Web Development. I have finished a few projects and am working on more! 
+          I am a hard worker and great with people.</p>
+      </section>
+      <section class="skills">
```
Comment (ari-mercado) [inline index.html:33]: This is purely preference, but some developers like to have an empty line separating each section, header, footer, ect. It just makes it a bit easier to read.

File: less/index.less [hunk 3]
```diff
@@ -54,6 +54,110 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
```
Comment (ari-mercado) [inline less/index.less:57]: Same as above, many devs believe having an empty line between each selector makes the code easier to read.

**Comment by ari7946** [comment]
Nice work, Isaac! great job with the table and the responsiveness works very well. My only suggestion is to commit more often. Looking forward to your future work!

**Pull request merged**